light-client/tests/model_based.rs

```diff
@@ -440,7 +443,7 @@ fn single_step_test(
         tc.initial.signed_header.clone(),
         tc.initial.next_validator_set.clone(),
     );
-    let clock_drift = Duration::from_secs(1);
+    let clock_drift = Duration::from_secs(0);
```

Comment:
Just wondering what's the purpose of this change? Does it strengthen the tests somehow by making them not rely on any compensation for the clock drift?

Reply:
We have disabled the clock drift, because the TLA+ model doesn't account for it at the moment. As a result, some tests were failing non-deterministically. As soon as the clock drift is introduced into the TLA+ model (@konnov and @josef-widder are working on that), we will also enable it here,

Reply:
Thanks for the clarification!

Reply:
The model does not account for `clock_drift` and so several timing-related tests were failing where the difference was of 1 second between a "valid" and "invalid" header. So it just made sense to remove this param from the test as well

Reply:
This PR introduces clock drift in the TLA+ spec: https://github.com/tendermint/spec/pull/216